Automattic/jetpack
# [RNMobile] Add Playback Panel to VideoPress Block Settings

Part of an ongoing project to add the VideoPress block to the native apps.

## Proposed changes:

* A native version of the `PlaybackPanel` component has been introduced and added to the VideoPress block's settings. The main code changes in the component's file can be found in https://github.com/Automattic/jetpack/pull/29457/commits/9f149cf9766ac0f2936d8d0d3615d4d137083917.

### Other information:


## Jetpack product discussion

p9ugOq-3mV-p2 

## Does this pull request change what data or activity we track or use?

No, it does not.

## Testing instructions:

* Go through the steps to add the VideoPress block on the web, optionally toggle the playback settings.
* Open up the app and navigate to the block that was added via the web.
* Tap the settings toggle to verify and then tap `Playback Settings` to open the new playback settings panel.
* If you'd changed the video's playback settings on the web, verify that these are accurately reflected on native.
* Experiment with toggling the settings and ensuring they work as expected.

### Screenshots

| 1st level  | 2nd level |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/225110777-c6345055-a82e-4efd-972c-6edcef152eb9.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/225110907-bee48ed9-f70c-4c8f-97cd-ed130d02ad51.png" width="100%"> |

### Screen recording

The following screen recording shows the extra helper text/warning that displays when the autoplay setting is enabled:

https://user-images.githubusercontent.com/2998162/225115829-3414c06b-facd-4586-870b-f8fa1897cd64.mov
